pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>io.snyk</groupId>
   <artifactId>snyk-maven-plugin</artifactId>
-  <version>1.2.2</version>
+  <version>1.3.2</version>
```

Comment:
shouldn't you change the version in the test pom.xml files as well?

Reply:
Also, this should be either `1.2.3` or `1.3.0`? Not `1.3.2`?

Reply:
yep sorry wrong place

Reply:
Updated